Added secret keys to .env

Phantsure opened the pull request:
Solved issue #15 
If anything more to be done in this file (.env) please point out

Phantsure committed:
Added secret keys to .env

JayjeetAtGithub commented on freepl_main/.env.example:
```diff
@@ -0,0 +1,2 @@
+SECRET_KEY=ajfbejqbt4j2b5jb4tkwjbkgjtbewkgjb
+DEBUG=True
```
```
SOCIAL_AUTH_FACEBOOK_KEY 
SOCIAL_AUTH_FACEBOOK_SECRET 
SOCIAL_AUTH_GOOGLE_OAUTH2_KEY
SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET
```

Add these variables to .env and set them with decouple in `settings.py.example`

Phantsure replied:
done

JayjeetAtGithub commented on freepl_main/settings.py.example:
```diff
@@ -12,6 +12,7 @@ https://docs.djangoproject.com/en/1.8/ref/settings/
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import os
+from decouple import config
```
Add `python-decouple` entry in `requirements.txt`

Phantsure replied:
done

JayjeetAtGithub reviewed (commented):
@Phantsure Please follow up with the review

Phantsure committed:
Update requirements.txt

Phantsure committed:
Update .env.example

added auth variables:
SOCIAL_AUTH_FACEBOOK_KEY 
SOCIAL_AUTH_FACEBOOK_SECRET 
SOCIAL_AUTH_GOOGLE_OAUTH2_KEY
SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET

Phantsure committed:
Update settings.py.example

Phantsure commented:
@JayjeetAtGithub please review again
